samples/apps/copilot-chat-app/webapi/Controllers/SpeechController.cs

```diff
@@ -0,0 +1,55 @@
+﻿
```

Comment:
I don't feel too hot about adding a controller to give out tokens in the SK service.

Talk to Teresa; she's working on something where she can request tokens from the frontend for various services and then pass them to the backend via a new header.

Also, more broadly, this feels like something that should simply be gated by AAD scopes.

I understand we're getting diverging requests and opinions about what the backend service and the frontend app are supposed to be and do.  It's becoming evident that we (the whole services and app crew) need to clarify this before going much further.

I'm afraid that by trying to "simplify" the backend service, we're actually making things more complicated!

Reply:
Resolved Offline; We need special admin privileges to make this possible, which isnt great for a reference implementation.